index.html

```diff
@@ -1,46 +1,176 @@
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
+
 <body>
     <header>
         <nav class="navbar">
             <p class="nav-item logo">Rb</p>
             <img class="nav-item" src="./img/menu.png" alt="the menu image ">
         </nav>
     </header>
+
+
     <section class="headline">
-        <div class="title">
+        <div class="head">
             <h2>Hey There.</h2>
-            <h2>I'm Rachid</h2> 
+            <h2>I'm Rachid</h2>
         </div>
         <p class="secondary">I'm a software developer</p>
-        <p class="details">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        <p class="details">I can help you build a product , feature or website Look through some of my work and
+            experience! If you like what you see and have a project you need coded,
             don’t hestiate to contact me.
-        
+
         </p>
         <div class="social-media">
-            <a href="https://github.com/benwmx" rel="noopener noreferrer" target="_blank"><img src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
-            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/" rel="noopener noreferrer" target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
-            <a href="https://twitter.com/benwmx" rel="noopener noreferrer" target="_blank"><img src="./img/003-twitter.png" alt="the image of twitter icon"></a>
+            <a href="https://github.com/benwmx" rel="noopener noreferrer" target="_blank"><img
+                    src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
+            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/" rel="noopener noreferrer"
+                target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
+            <a href="https://twitter.com/benwmx" rel="noopener noreferrer" target="_blank"><img
+                    src="./img/003-twitter.png" alt="the image of twitter icon"></a>
         </div>
     </section>
-       
-    
+
+
+    <section class="work">
+        <h1>My Recent Works</h1>
+        <div class="line1">
+        </div>
+        <div class="card one">
+            <h1 class="title">Professional Art <br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard</p>
+            <div class="languages-built">
+                <p>Html</p>
+                <p>Bootstrap</p>
+                <p>Ruby</p>
+            </div>
+            <button type="button" class="button project">
+                See Project
+            </button>
+        </div>
```

Comment:
- [ ] You used `divs` in grouping the list of projects. For cleaner syntax let's use a `ul` tag to group them, and `li` per project.

Reply:
in the instructions, it says to use Grid to lay the cards 
although I don't get the purpose of using list here, did change the code as you requested

Reply:
**Hi Rachid!**

Here's the guide, point number 5:
![image](https://user-images.githubusercontent.com/12745474/125077291-d8277180-e0b8-11eb-86e4-0a75f3eedf3f.png)

In case this was not listed as part of the requirements you used for this project then that's a mistake on our side, and we will ensure to rectify that.

I'm sorry I couldn't figure out this issue during my review. Kindly note, this was not on purpose 😊 

Happy coding! 🍷 💻 

Cheers!